### DISCUSSION TITLE
Issue with updating stream with cluster name

### DISCUSSION THREAD

**Discussion by pcsegal** [nats-io/nats-server#5822] [Q&A]
Hi,

I have a development NATS cluster with 8 pods in Kubernetes (server version 2.10.17). I've got a situation where I tried to update a KV bucket's underlying stream (R=1) to add a cluster name to it, using `update stream <stream name> --cluster <cluster name>` (using the same cluster name as it was before, just for testing purposes). It appears to be trying to move the stream to another peer of the cluster, but is stuck. If I do a `nats kv info <kv_name>`, I get:

```
Leader:
Replica: Server name unknown at this time (peerID: Mbbz6gHk), current, OFFLINE, not seen
Replica: <replica name>, outdated, seen 2m21s ago
```

The stream was fine before trying to update it. If I try to do any other update to this stream, I get the error that a move is already in progress. If I try to run some other command like `stream view` on the stream, I get context deadline exceeded.

The issue starts happening after I try to update the stream as described.
The cluster peers seem to be current and active according to `server request jetstream`.

The logs show that this stream is stalled (no quorum).

**Comment by derekcollison** [comment]
What server version? 

You can cancel a move, or scale down to an R1 and back up to R3.

**Reply by pcsegal** [reply]
Server version is 2.10.17. In this case the stream is R1.

**Reply by derekcollison** [reply]
<img width="635" alt="image" src="https://github.com/user-attachments/assets/ec60078e-bb45-4741-9edf-87686545c596">

Request needs to be sent from system account. The two trailing tokens are account and stream name.

**Reply by pcsegal** [reply]
Thank you, I went back to this today and tried cancelling the move with the suggested API, and the problematic stream / KV is now back online.
